src/Inciter/ALECG.cpp

```diff
@@ -139,7 +139,8 @@ ALECG::ALECG( const CProxy_Discretization& disc,
   thisProxy[ thisIndex ].wait4lhs();
 
   // Signal the runtime system that the workers have been created
-  contribute( sizeof(int), &m_initial, CkReduction::sum_int,
+  std::vector< std::size_t > meshdata{ m_initial, d->MeshId() };
+  contribute( meshdata, CkReduction::sum_ulong,
```

Comment:
In this case, the target of the reduction is `Transporter::comfinal()` which is used to signal that a task (computing communication maps) is finished on all ALECG chares and on the receiving side, `comfinal` is not part of any SDAG. The additional data that we are reducing, `m_initial`, signals _where_ we are calling `comfinal` from (within a time step): during setup (initial == 1) or during time stepping (initial == 0). The second argument collected, `meshid`, is used to signal which mesh the reduction is coming from, i.e., which (part of coupled) physics is done with this task (on its mesh). So since there is the additional data (`initial`) to be collected and since comfinal is not part of an SDAG in the target (Transporter) can I still specify meshid as a refnum somehow?

Reply:
No, refnums are only for use in SDAG. So yeah, I think most of my comments from the review are irrelevant until there is SDAG usage. Right now there is not honestly a great way to send metadata like this along with a reduction. The only other thing you could do would be a tuple reduction, which allows you to specify multiple operations. Sum for the m_initial, and nop for meshid. But tuple reductions are a pain to use so I generally avoid them.

Reply:
Okay, that's not a problem. Thanks for teaching me about this stuff!